theme/management/commands/email_users.py

```diff
@@ -0,0 +1,38 @@
+from django.core.management.base import BaseCommand
+from django.core.mail import send_mail
+from django.conf import settings
+from django.contrib.auth.models import User
+
+from theme.models import UserQuota, QuotaMessage
+
+
+class Command(BaseCommand):

```

Comment:
@pkdash This management command is supposed to be invoked by @mjstealey's nightly-run script that calculates quota overnight and assembles all user names whose quotas have exceeded the allocated quotas, then calls this management command to pass in all these user names and this management command will send overquota emails to all passed in user names. Does this make sense? Let me know if you have any further questions.